hw-L06-reflection-test-runner/src/main/java/com/abrenchev/After.java

```diff
@@ -0,0 +1,11 @@
+package com.abrenchev;
```

Comment:
```suggestion
package com.abrenchev.annotations;
```

Аннотации можно в отдельный пакет.

Reply:
Done.